### PR TITLE
Fixing doc to remove github app requirement

### PR DESCRIPTION
**Pull request by rshade**

### CODE REVIEW

File: content/docs/pulumi-cloud/admin/self-hosted/components/deployments.md [hunk 1]
```diff
@@ -28,8 +28,6 @@ To manage your state with a self-managed backend, such as a cloud storage bucket
 
 To do so, follow these steps:
 
-1. [Install the Pulumi GitHub app](https://www.pulumi.com/docs/iac/packages-and-automation/continuous-delivery/github-app) and ensure that it's integrated with your source control.
-
 1. Create a new deployment pool. Navigate to **Settings > Deployment runners** in the left-hand menu of your Pulumi Cloud organization, click **Add new pool**, and copy the generated access token.
```
Comment (interurban) [inline content/docs/pulumi-cloud/admin/self-hosted/components/deployments.md:31]:
```suggestion
1. Create a new deployment pool. Navigate to **Settings** then **Deployment runners** in the left-hand menu of your Pulumi Cloud organization, click **Add new pool**, and copy the generated access token.
```

Comment (interurban) [reply content/docs/pulumi-cloud/admin/self-hosted/components/deployments.md:31]: Thank you @rshade  !

**Review by interurban — APPROVED** [review]
One small fix up; then you are ready to go

**Comment by rshade** [comment]
Awesome, thanks @interurban

**Pull request merged**